mackup/applications/github-cli.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = GitHub CLI
+
+[configuration_files]
+.config/gh/config.yml
```

Comment:
Please see https://github.com/lra/mackup/blob/master/doc/README.md#add-support-for-an-application-using-the-xdg-directory